src/main.cpp

```diff
@@ -731,7 +731,7 @@ int main ( int argc, char** argv )
 \******************************************************************************/
 QString UsageArguments ( char** argv )
 {
-    return "Usage: " + QString ( argv[0] ) +
+    return "\nUsage: " + QString ( argv[0] ) +
            " [option] [optional argument]\n"
            "\nGeneral options:\n"
```

Comment:
I'd rather the extra `\n`s were on their own lines - it's easier to see the resultant layout, that way.

Reply:
While I add this, are you okay with updating the following lines:

Current Line:
```  -e, --directoryserver address of the directory server with which to register```
Proposed New Line:
```  -e, --directoryserver address of the directory server to register with```

Current Line:
```
  -i, --inifile         initialization file name (not
                        supported for headless server mode)
```

Proposed New Line:
```
  -i, --inifile         initialization file name
                        (not supported for headless server mode)
```

Reply:
> -e, --directoryserver  Proposed New Line:

I prefer the correct English current version :).

> -i, --inifile Proposed New Line:

Nicer layout, yes.